client/jetpack-cloud/sections/partner-portal/issue-license-v2/licenses-form/index.tsx

```diff
@@ -36,6 +41,8 @@ export default function LicensesForm( {
 		PRODUCT_FILTER_ALL
 	);
 
+	const { data } = useProductsQuery();
```

Comment:
Can't we just call `useProductBundleSize` hook here and get `availableSizes` instead of exposing `getSupportedBundleSizes` and retrieving data here?